robots/solo_description/robots/solo12.urdf

```diff
@@ -9,15 +9,15 @@
     <!-- BASE LINK INERTIAL -->
     <inertial>
       <origin rpy="0 0 0" xyz="0 0 0"/>
-      <mass value="1.63315091"/>
+      <mass value="1.16115091"/>
       <!-- The base is extremely symmetrical. -->
       <inertia ixx="0.00578574" ixy="0.0" ixz="0.0" iyy="0.01938108" iyz="0.0" izz="0.02476124"/>
     </inertial>
     <!-- BASE LINK VISUAL -->
     <visual>
       <origin rpy="0 0 0" xyz="0 0 0"/>
       <geometry>
-        <mesh filename="package://solo_description/meshes/obj/with_foot/solo_body.obj"/>
+        <mesh filename="package://solo_description/meshes/obj/solo12/solo_12_base.obj"/>
```

Comment:
We should update all these paths to `package://example_robot_data/robots/solo_description/...`

Reply:
I agree with this, but this is a separate issue: #19

Reply:
Shouldn't we address them directly when adding new URDFs though? When touching Solo, we should in the same step fix its URDFs - don't you agree? Instead of doing a separate action for that immediately after.

Reply:
I am not adding a new URDF here.

I think this might be harder to track changes about one issue if the similar commits are everywhere. 

I also think the state of this repository should be consistent.
The path as they are now are broken for some use case, but working for others. Right now, universal-robot's URDF are not like the other ones, meaning that all tests are passing except those on UR.

If we make this change here, all tests will pass except those on UR & solo 12.

If you suggest to update solo 12, while not updating solo  8, it's going to be non-consistent even in the same robot directory. And if you suggest to update solo 8 and solo  12, this change a bit the scope of this PR.

I think the best option here is to fix #19 first, then rebase this PR on that fix.

Reply:
> If you suggest to update solo 12, while not updating solo 8, it's going to be non-consistent even in the same robot directory. And if you suggest to update solo 8 and solo 12, this change a bit the scope of this PR.

My suggestion was to update both Solo URDFs. 

Either way, I think the point is we should fix #19.

Reply:
done :)